examples/enums/enums.description

```diff
@@ -0,0 +1,4 @@
+// Enums provides a convenient syntax for declaring a union of string constants.
```

Comment:
shouldn't this be `An Enum`

Reply:
Other examples have used a similar approach:
For example, Constants : https://github.com/ballerina-platform/ballerina-distribution/blob/master/examples/constants/constants.description

@MaryamZi @pubudu91 WDYT?

Reply:
Is "Enums" plural or singular.. then we need to change `provides`.
Sorry for not being clear?

Reply:
Yeah, should be provide

Reply:
+1

Reply:
+1 for provide